test/python/WMCore_t/Services_t/Dashboard_t/DashboardAPI_t.py

```diff
@@ -34,22 +32,23 @@ def testApmonInstance(self):
 
         Just test initialization of apmon Instance
         """
-        print("Apmon Configuration %s" % APMONCONF)
-        apmon = getApmonInstance(apmonServer=APMONCONF)
-        self.assertTrue(apmon.initializedOK())
+        dashboard = DashboardAPI()
```

Comment:
This pattern looks an awful lot like something that could be done as a context manager. Maybe we should think about adding those methods to DashboardAPI